Marlin/pins.h

```diff
@@ -1309,6 +1315,7 @@
        #endif
      #else
        #define BTN_ENC 16  //the click switch
+       #define LCD_SDSS 28 //to use the SD card reader on the smart controller rather than the melzi board
```

Comment:
Is this something that all smart-controller users have been needing, but it was left out until now?